rasa/core/training/generator.py

```diff
@@ -727,6 +727,9 @@ def _issue_unused_checkpoint_notification(
         collected_start = set()
         collected_end = set()
         for step in self.story_graph.story_steps:
+            if step.is_rule:
```

Comment:
fix most of the warnings with this hack 👀

Reply:
but `md` format doesn't require this hack? I think there is smth wrong with yaml then

Reply:
no, both require this hack.

Reply:
wait, isn't this hack disables the warning for all non rule stories? that's not a good idea